app.js

```diff
@@ -72,10 +73,30 @@ if (process.env.NODE_ENV !== 'production') {
   require('./webpack/webpack')(app);
 }
 
+// get the base js filename
+const publicJsPath = `${__dirname}/public/js`;
+let baseJsFile;
+if (process.env.NODE_ENV === 'production') {
+  const files = fs.readdirSync(publicJsPath);
+  console.log(files);
+  files.sort((val1, val2) => {
+    const stat1 = fs.statSync(`${publicJsPath}/${val1}`);
+    const stat2 = fs.statSync(`${publicJsPath}/${val2}`);
+    return stat2.mtime - stat1.mtime;
+  });
+  const reg = /^app.min.[\w]+.js$/;
+  baseJsFile = files.find(f => reg.test(f));
+}
+if (baseJsFile === undefined) {
+  baseJsFile = 'app.min.js';
+}
+logger.info(`The latest base js file is: ${baseJsFile}`);
```

Comment:
the same as above, if this is not a must, then remove it

Reply:
This is planned.